includes/class-seo-data-transporter.php

```diff
@@ -0,0 +1,315 @@
+<?php
+/**
+ * SEO Data Transporter
+ *
+ * @package seo-data-transporter
+ */
+
+/**
+ * The SEO Data Transporter main class.
+ *
+ * @since 1.0.0
+ */
+final class SEO_Data_Transporter {
+
+	/**
+	 * Plugin version
+	 *
+	 * @var $plugin_version Plugin Version
+	 */
+	public $plugin_version = '1.1.0';
```

Comment:
@ethan-schaffer I would be nice to define a new constant for the version (like we've been doing for other plugins). It could be `define( 'SEO_DATA_TRANSPORTER_VERSION', '1.1.0' );` in the main plugin file, and here, we could have something like:


```suggestion
	public $plugin_version = SEO_DATA_TRANSPORTER_VERSION;
```
This would avoid us needing to look other files to update the plugin version on releases :)